### ISSUE TITLE
请问CTC对数概率log_prob可以转换为车牌的识别率么？

### ISSUE THREAD

**Issue by huchenw** [lyl8213/Plate_Recognition-LPRnet#23]
源文件LPRtf3.py的343行，log_prob有什么用？请问如何输出车牌的识别准确率呢？谢谢！
`decoded, log_prob = tf.nn.ctc_beam_search_decoder(logits, seq_len, merge_repeated=False)`
